skyportal/handlers/photometry.py

```diff
@@ -68,6 +71,23 @@ def post(self):
             DBSession().add(p)
             DBSession().flush()
             ids.append(p.id)
+        if 'thumbnails' in data:
+            for thumb in data['thumbnails']:
+                file_uri = os.path.abspath(
+                    f'static/thumbnails/{source.id}_{thumb["type"]}.png')
+                file_bytes = base64.b64decode(thumb['data'])
+                im = Image.open(io.BytesIO(file_bytes))
+                if im.format != 'PNG':
+                    return self.error('Invalid thumbnail image type. Only PNG are supported.')
+                if im.size != (200, 200):
```

Comment:
Genuine question: how restrictive do we want to be with thumbnails?  Perhaps a range check here of between 100 and 500?

Reply:
This should also be documented in the API docs.